client/src/questions-n-answers/sub-components/questionContainer.jsx

```diff
@@ -6,22 +6,25 @@ import QuestionList from './mini-components/questionList.jsx';
 const QuestionsContainer = (props) => {
 
   let showQuestionClass = props.showQuestions(props.questionClickCount, props.currentI)
-  let show = false;
-  if (props.questionClickCount === 1 && props.currentI <= props.questionClickCount) {
-    show = true
-  }
-  if (props.questionClickCount >= 3 && props.currentI<= props.questionClickCount) {
-    show = true
-  }
+  // let show = false;
```

Comment:
Is this commented out because it is work in progress?

Reply:
no sir, needs to be removed - moved the functionality to another location thanks!

Reply:
👍